ivy/functional/frontends/torch/__init__.py

```diff
@@ -3,3 +3,5 @@
 from .indexing_slicing_joining_mutating_ops import *
 from . import pointwise_ops
 from .pointwise_ops import *
+from . import Utilities
```

Comment:
You'll need to change utilities to lowercase pending the changes to the utilities file